### PR TITLE
Kernel: Even more stubs for AARCH64

### PR DESCRIPTION
**Pull request by gunnarbeutner**
With these changes a lot more files (including the scheduler) are buildable.

### CODE REVIEW

File: Kernel/Arch/aarch64/ASM_wrapper.h [hunk 1]
```diff
@@ -92,5 +92,13 @@ inline void enter_el1_from_el2()
...
                      : "x0");
 }
+}
+
+namespace Kernel {
+inline bool are_interrupts_enabled()
+{
+    auto daif = Aarch64::DAIF::read();
+    return !daif.I;
+}
```
Comment (FireFox317) [inline Kernel/Arch/aarch64/ASM_wrapper.h:102]: I've been using `Processor::are_interrupts_enabled` instead, so you could replace the callsites of this function with `Processor::are_interrupts_enabled`, or call `Processor::are_interrupts_enabled()` in this function.

Comment (gunnarbeutner) [reply Kernel/Arch/aarch64/ASM_wrapper.h]: 👍

**Review by ADKaster — APPROVED** [review]
Nothing looks particularly sketchy in these patches, though I do wonder if the stubbed Thread and Process functions *really* need to be arch specific

File: Kernel/Arch/aarch64/Dummy.cpp [hunk 2]
```diff
@@ -139,6 +142,25 @@ Thread::Blocker::~Blocker()
...
+SpinlockProtected<Thread::GlobalList>& Thread::all_instances()
+{
+    TODO_AARCH64();
+}
+
+Thread::~Thread()
+{
+    TODO_AARCH64();
+}
 }
```
Comment (linusg) [inline Kernel/Arch/aarch64/Dummy.cpp:186]: Newline around namespaces

**Review by linusg — CHANGES_REQUESTED** [review]
> Kernel: More stubs for Thread

https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy
> Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").

File: Kernel/Arch/aarch64/ASM_wrapper.h [hunk 3]
```diff
@@ -92,5 +93,12 @@ inline void enter_el1_from_el2()
                  "entered_el1:" ::
                      : "x0");
 }
+}
```
[the remark this answers is not in the document]
Comment (gunnarbeutner) [reply Kernel/Arch/aarch64/ASM_wrapper.h:96]: 👍

File: Kernel/Arch/aarch64/ASM_wrapper.h [hunk 4]
```diff
@@ -92,5 +93,12 @@ inline void enter_el1_from_el2()
...
+
+namespace Kernel {
 
+inline bool are_interrupts_enabled()
+{
+    return Processor::are_interrupts_enabled();
+}
```
[the remark this answers is not in the document]
Comment (gunnarbeutner) [reply Kernel/Arch/aarch64/ASM_wrapper.h:103]: 👍

File: Kernel/Arch/aarch64/Dummy.cpp (hunk 2 above)
Comment (gunnarbeutner) [reply Kernel/Arch/aarch64/Dummy.cpp:186]: 👍

**Pull request merged**